src/style.css

```diff
@@ -1,3 +1,93 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
You should avoid using the '*' selector as it impacts performance. It's better to set styles only for elements that need them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for the browser.